Source/AppAuthTV/OIDTVServiceConfiguration.m

```diff
@@ -36,18 +35,16 @@ - (nullable instancetype)initWithCoder:(NSCoder *)aDecoder NS_DESIGNATED_INITIAL
 
 @end
 
-@implementation GTMTVServiceConfiguration
-
-@synthesize TVAuthorizationEndpoint = _TVAuthorizationEndpoint;
+@implementation OIDTVServiceConfiguration
 
 - (instancetype)init
     OID_UNAVAILABLE_USE_INITIALIZER(
-        @selector(initWithAuthorizationEndpoint:TVAuthorizationEndpoint:tokenEndpoint:));
+                                    @selector(initWithAuthorizationEndpoint:TVAuthorizationEndpoint:tokenEndpoint:))
```

Comment:
[Style-guide is indent](https://google.github.io/styleguide/objcguide.html) 4 spaces when a line break is needed. I htink either it should be indented 4 spaces here or the linebreak removed completely (if it fits).